### DISCUSSION TITLE
[Style] position sticky don't work in GrapesJS

### DISCUSSION THREAD

**Discussion by robin59161** [GrapesJS/grapesjs#3895] [Q&A]
Hello, i have an element which have the position : sticky attributes in the css, but in grapesjs, this property doesn't work, do you know if it has a way to do this?

**Comment by artf** [comment]
You have to indicate on initialization [what properties you need](https://grapesjs.com/docs/getting-started.html#style-manager).

**Reply by robin59161** [reply]
Okay, i've done that, but my button doesn't show sticky in grapesJs but in css rules it has 'position: sticky'

**Reply by artf** [reply]
Please create a reproducible demo so we can see what is happening

**Reply by robin59161** [reply]
Here a JsFiddle, my sticky cta (custom component) doesn't look at a sticky position : https://jsfiddle.net/0pyqudt9/

**Reply by artf** [reply]
Are you sure you're selecting the right component??  I see it from your demo...

**Reply by robin59161** [reply]
![image](https://user-images.githubusercontent.com/73250776/140749684-7172ee9a-6d89-45fa-b5dc-07634f32398f.png)
In picture is what i need to be sticky, the style manager have the correct sticky position and same for the css in the interpretor
![image](https://user-images.githubusercontent.com/73250776/140749735-37bab3ca-3046-46c0-af08-79eeae57f489.png)
